LAYOUT_STYLE_GUIDE.md

```diff
@@ -170,30 +158,34 @@ Use Heading 3 `###` and [Wikipedia Style Title Case](https://titlecaseconverter.
 
 Use the [Title Case Converter](https://titlecaseconverter.com/) tool to help convert text to Title Case, being sure to select the Wikipedia "Styles" option.
```

Comment:
I know there are a couple of links here, but would it also be helpful to actually give an example that shows what Wikipedia Title Case is and is not?

Alternately, could also break up the line above on "Use Heading 3 `###` and Wikipedia Style Title Case for main section titles:" so that we give some emphasis to the note on using Wiki's Title Case by putting it on its own line.

Reply:
Example is tough since a lot of title cases do end up the same and it's more minor differences, but I provided a more drastic "Wiki vs non-Wiki style" example.

Reply:
Nice, looks good!